pkg/monitor/monitor/monitor_manager.go

```diff
@@ -75,12 +75,24 @@ func (m *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 	}
 
 	var firstTc *v1alpha1.TidbCluster
+	assetStore := NewStore(m.deps.ConfigMapLister, m.deps.SecretLister)
+
 	for _, tcRef := range monitor.Spec.Clusters {
 		tc, err := m.deps.TiDBClusterLister.TidbClusters(tcRef.Namespace).Get(tcRef.Name)
 		if err != nil {
 			rerr := fmt.Errorf("get tm[%s/%s]'s target tc[%s/%s] failed, err: %v", monitor.Namespace, monitor.Name, tcRef.Namespace, tcRef.Name, err)
 			return rerr
 		}
+
+		// If cluster enable tls
+		if tc.IsTLSClusterEnabled() {
+			tcTlsSecretName := util.ClusterClientTLSSecretName(tc.Name)
+			err := assetStore.addTLSAssets(tc.Namespace, tcTlsSecretName)
+			if err != nil {
+				return err
```

Comment:
Can we just continue to sync for other clusters? For the error here, we can choose one of the solutions:
1. print error log and continue, wait for the next resync period (30s)
2. aggrate and return the error in the end of this function

Reply:
I can print log,but this for loop code just returns now when it encounters an error ,you can look at the context around the code.